Starscream.podspec

```diff
@@ -11,11 +11,10 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = '10.10'
   s.tvos.deployment_target = '9.0'
   s.watchos.deployment_target = '2.0'
-  s.source_files = 'Sources/*.swift'
-  s.libraries    = 'z'
+  s.source_files = 'Sources/**/*.{h,m,swift}'
+  s.module_map = 'Sources/modulemap/Starscream.modulemap'
+  s.private_header_files = 'Sources/modulemap/**/*.h'
```

Comment:
If we delete modulemap and header file then this is not needed anymore.

Test also with some sample project integrating this pod against https://websocket.org/echo.html